src/main/java/com/joyent/manta/monitor/MantaMonitorJerseyModule.java

```diff
@@ -11,34 +11,40 @@
 import com.google.inject.Injector;
 import com.google.inject.Provider;
 import com.google.inject.servlet.ServletModule;
+import com.joyent.manta.client.MantaClient;
 import io.logz.guice.jersey.JettyServerCreator;
 import io.logz.guice.jersey.configuration.JerseyConfiguration;
 import java.util.Objects;
 import org.eclipse.jetty.server.Server;
 
 /**
- * This implementation of {@link AbstractModule} provides dependency injection for the creating and configuring
- * {@link MantaMonitorJerseyServer}.
+ * This implementation of {@link AbstractModule} provides dependency injection
+ * for the creating and configuring {@link MantaMonitorJerseyServer}.
  */
 
 public class MantaMonitorJerseyModule extends AbstractModule {
     private final JerseyConfiguration jerseyConfiguration;
     private final JettyServerCreator jettyServerCreator;
+    private final MantaClient client;
 
-    public MantaMonitorJerseyModule(final JerseyConfiguration jerseyConfiguration) {
-        this(jerseyConfiguration, Server::new);
+    public MantaMonitorJerseyModule(final JerseyConfiguration jerseyConfiguration,
+                                    final MantaClient client) {
+        this(jerseyConfiguration, Server::new, client);
     }
 
-    public MantaMonitorJerseyModule(final JerseyConfiguration jerseyConfiguration, final JettyServerCreator jettyServerCreator) {
+    public MantaMonitorJerseyModule(final JerseyConfiguration jerseyConfiguration,
```

Comment:
I think the dependency injection configuration can be simplified, but I will hold back on my suggestions until the other issues have been addressed.

Reply:
[This](https://github.com/1010sachin/manta-monitor-1/compare/keystore_path_as_uri...dekobon:keystore_path_as_uri) is what I mean about simplification. Refactor it so something simpler like that.

Reply:
Merged the suggested changes.